actix-http/src/body/body.rs

```diff
@@ -33,39 +36,70 @@ impl AnyBody {
         Self::Bytes(Bytes::new())
     }
 
-    /// Create body from slice (copy)
+    /// Create boxed body from generic message body.
+    pub fn new_boxed<B>(body: B) -> Self
```

Comment:
Maybe just call this `boxed`?

Reply:
toyed wth the idea yea

the counter argument is that `boxed` is well known as a combinator-style method in futures-util: https://docs.rs/futures-util/0.3.17/futures_util/future/trait.FutureExt.html#method.boxed

Reply:
Im sure counter examples exist. This shouldn't get used too much i think. I just like the idea of being explicit about its function and that `new` and `new_boxed` would be together in the docs.rs sidebar.